andela/lotus-ah-frontend
# #160451556 Users should be able Signup/Register to Authors Haven

#### What does this PR do?
Implement user signup with authentication

#### Description of Task to be completed?
- create signup, verify email, and update pages
- create signup, verify email and update pages container
- create signup, verify email, and update action types
- create signup, updateAccount, and activate account reducers with initial states

#### How should this be manually tested?
```steps```
- start the app
- click on get started on the landing the page
- on the next page select signup with email
- click signup after entering your email on a modal that will pop up

- go to your mailbox to activate your account
- update your details
- you should be redirected to the home page with your username on the header

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[Delivers #160451556]

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-08 at 1 33 25 am" src="https://user-images.githubusercontent.com/24890605/48170415-bb2eb500-e2f7-11e8-9be0-a08dfc7672f2.png">

<img width="1440" alt="screen shot 2018-11-08 at 1 33 44 am" src="https://user-images.githubusercontent.com/24890605/48170437-d3063900-e2f7-11e8-94ba-ec4b2b44a0ff.png">

<img width="1440" alt="screen shot 2018-11-08 at 1 34 16 am" src="https://user-images.githubusercontent.com/24890605/48170470-f5985200-e2f7-11e8-803c-f9425f6f45b6.png">



#### Questions:
